src/main/java/org/apache/ibatis/scripting/xmltags/XMLScriptBuilder.java

```diff
@@ -88,17 +103,7 @@ public SqlSource parseScriptNode() {
   }
 
   NodeHandler nodeHandlers(String nodeName) {
-    Map<String, NodeHandler> map = new HashMap<String, NodeHandler>();
-    map.put("trim", new TrimHandler());
-    map.put("where", new WhereHandler());
-    map.put("set", new SetHandler());
-    map.put("foreach", new ForEachHandler());
-    map.put("if", new IfHandler());
-    map.put("choose", new ChooseHandler());
-    map.put("when", new IfHandler());
-    map.put("otherwise", new OtherwiseHandler());
-    map.put("bind", new BindHandler());
-    return map.get(nodeName);
+    return nodeHandlerMap.get(nodeName);
```

Comment:
I would remove this method and call `nodeHandlerMap.get()` directly.
It's redundant and the name is inaccurate.

Reply:
Good idea.